gnssanalysis/gn_download.py

```diff
@@ -703,7 +671,7 @@ def download_file_from_cddis(
     :param str if_file_present: What to do if file already present: "replace", "dont_replace", defaults to "prompt_user"
     :param str note_filetype: How to label the file for STDOUT messages, defaults to None
     :raises e: Raise any error that is run into by ftplib
-    :return _Path: Return the local Path of the file downloaded
+    :return _Path: Return the pathlib.Path of the downloaded file
```

Comment:
```suggestion
    :return _Path: The pathlib.Path of the downloaded file
```
Very minor phrasing detail, it just sounds less redundant dropping the second 'return'

Reply:
No worries, removed the `Return` here and the other 3 places where this appears